locales/fr_FR/LC_MESSAGES/duckduckgo.po

```diff
@@ -20,32 +20,35 @@ msgstr ""
 "X-Generator: Poedit 2.3\n"
 
 msgid "!Bang Search Shortcuts"
-msgstr "Raccourcis !Bang"
+msgstr "Raccourcis de recherche !Bang"
 
+#. input correct way to showcase amounts in French (done)
```

Comment:
These comments don't belong in the file. If you want to specify why you made a change, you can add comments to the commit message or better still commit the file and add comments in the PR, like I am doing now. Please remove all these comments. Thanks for your help! 🙇 
```suggestion
```